### PR TITLE
Linting ``nextcord``

### PR DESCRIPTION
**Pull request by VincentRPS**
## Summary

Linted nextcord to be more readable; And Fully follow pep8

## Checklist

- [x] If code changes were made then they have been tested.

### CODE REVIEW

**Comment by ooliver1** [comment]
mmmm not sure about this now, linting code you did not make can cause issues

File: .gitignore
Comment (ooliver1) [inline .gitignore:22]: like what are mp4's doing here

Comment (VincentRPS) [reply .gitignore:22]: @ooliver1 I remember there was a conversation to support mp4 or not in voice rec just added is Incase

Comment (ooliver1) [reply .gitignore:22]: Generally you don't find people storing the mp4's in the same dir as the repo, anyway if you're bothering to do that there's a lot more formats

Comment (ooliver1) [reply .gitignore:22]: Oh also keep PR's to one issue/topic maybe

Comment (VincentRPS) [reply .gitignore:22]: Ah Okay

File: .gitignore
Comment (TAG-Epic) [inline .gitignore:17]: should not be changed?

File: aliasgen.py
Comment (TAG-Epic) [inline aliasgen.py:43]: this looks awful, are you sure this isn't a linter bug?

File: aliasgen.py
Comment (TAG-Epic) [inline aliasgen.py:82]: why replace this?

File: discord/abc.py
Comment (TAG-Epic) [inline discord/abc.py:30]: aliasgen doesn't generate linted scripts? Maybe add that as a step?

File: docs/conf.py
Comment (TAG-Epic) [inline docs/conf.py:93]: this looks awful.

File: docs/conf.py
Comment (TAG-Epic) [inline docs/conf.py:27]: comments are half linted here where, it uses single instead of double quotes in comments?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
I honestly think we should wait until voice receive and slash commands are merged then make a linting CI and setup.py task imo

File: discord/abc.py
Comment (VincentRPS) [reply discord/abc.py:30]: hm?

**Comment by VincentRPS** [comment]
actually i agree with epic to make this a cd/ci process so i am gonna close this

**Pull request closed without merging**
